fastify.js

```diff
@@ -464,6 +459,11 @@ function build (options) {
         return
       }
 
+      // run 'onRoute' hooks
```

Comment:
I'm not sure about this change, if the `onRoute` hook changes something in the route declaration what will happen?

Reply:
You're right. How to deal with route errors?

Reply:
What's your expectation? That any `onRoute` route is valid or that just to announce that a route was registered?